nix/modules/k3s.nix

```diff
@@ -39,8 +39,19 @@ in
     services.k3s.enable = true;
     services.k3s.docker = false;
 
-    # for some reasons our firewall, breaks caligo... no body got time to debug this...
-    networking.firewall.enable = false;
+    # example firewall:
+    networking.firewall.enable = true;
+    networking.firewall.allowedTCPPortRanges = [ { from = 0; to = 65535; } ];
```

Comment:
Since everything is allowed what does this fix?

Reply:
The reverse path might have been indeed the problem.

Reply:
having it enabled allows adding further firewall.extraCommands which i use to make specific restrictions.